packages/sdk/src/rpcCommon.ts

```diff
@@ -1,25 +1,32 @@
 import { Interceptor, Transport } from '@connectrpc/connect'
-import {
-    createConnectTransport as createConnectTransportWeb,
-    ConnectTransportOptions as ConnectTransportOptionsWeb,
-} from '@connectrpc/connect-web'
+import { type ConnectTransportOptions as ConnectTransportOptionsWeb } from '@connectrpc/connect-web'
 import { type RetryParams } from './rpcInterceptors'
-import { isNodeEnv, isTestEnv } from '@towns-protocol/dlog'
+import { isNodeEnv, isBrowser } from '@towns-protocol/dlog'
 
 export interface RpcOptions {
     retryParams?: RetryParams
     interceptors?: Interceptor[]
 }
 
-export function createHttp2ConnectTransport(options: ConnectTransportOptionsWeb): Transport {
-    if (isNodeEnv && !isTestEnv()) {
-        // use node version of connect to force httpVersion: '2'
-        const {
-            createConnectTransport: createConnectTransportNode,
-            // eslint-disable-next-line import/no-extraneous-dependencies, @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
-        } = require('@connectrpc/connect-node')
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-        return createConnectTransportNode({ ...options, httpVersion: '2' }) as Transport
+export async function createHttp2ConnectTransport(
+    options: ConnectTransportOptionsWeb,
+): Promise<Transport> {
+    try {
+        if (isNodeEnv) {
```

Comment:
the previous code ensured that we use the browser version in the test suite - which might be why the tests are now failing?